pkgs/top-level/all-packages.nix

```diff
@@ -2566,8 +2566,15 @@ with pkgs;
 
   microcom = callPackage ../applications/terminal-emulators/microcom { };
 
-  mlterm = darwin.apple_sdk_11_0.callPackage ../applications/terminal-emulators/mlterm {
-    inherit (darwin.apple_sdk_11_0.frameworks) Cocoa;
+  mlterm = callPackage ../applications/terminal-emulators/mlterm { };
+  mlterm-wayland = mlterm.override {
+    guiChoices = {
+      wayland = true;
+    };
```

Comment:
Why does this not appear in the GUI choice list? Why isn't this on by default?

Wouldn't it make more sense to have all GUI toolkits enabled in regular mlterm make mlterm-wayland mlterm without xorg?

Reply:
I agree, wayland should be enabled by default, along with all other gui options.

Reply:
My original intention was not to change this package in terms of what's actually get compiled, and work only on the expression.